src/actions/action.ts

```diff
@@ -117,6 +119,7 @@ export function createAction(item: ActionSpec): Action {
     isReadonlyAllowed: item.isReadonlyAllowed || false,
     separator: item.separator || false,
     icon: typeof icon === "function" ? icon : () => icon || "",
+    iconColor: item.iconColor,
```

Comment:
i don't really like this. we have icon, secondaryIcon, now iconColor. Just a matter of time before we have secondaryIconColor? shouldn't the icon key be changed to be an object {template, color} ? Or (though we probably already thought about it) having icon return a wrapper that adds a color to the the icon template?

```javascript
icon: getColoredTemplate('myString', 'red')
...
function getColoredTemplate(str, color) {
returns ```
<div style="color:{color}">
<t-call={str} />
</div>
```
might not be feasible without adding some injection possibilities?

Reply:
yeah, it would require to bypass escaping and open the door to injections. And in this case, the color is user provided, so definitely not on the safe side